firefox-ios/Client/Frontend/Home/HistoryHighlights/HistoryHighlightsCell.swift

```diff
@@ -131,6 +132,19 @@ class HistoryHighlightsCell: UICollectionViewCell, ReusableCell {
         ])
     }
 
+    override public func layoutSubviews() {
+        super.layoutSubviews()
+        let size: CGFloat = 5
+        let distance: CGFloat = 0
+        let rect = CGRect(
+            x: -size,
+            y: contentView.frame.height - (size * 0.4) + distance,
+            width: contentView.frame.width + size * 2,
+            height: size
+        )
+        contentView.layer.shadowPath = UIBezierPath(ovalIn: rect).cgPath
```

Comment:
Shouldn't this code only be added if it's a cell that needs a shadow?

Reply:
Hey, thanks for pointing it out, have updated the PR with a fix. Please have a look.